scripts/use_node

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+#
+
+# This script will find the appropriate Node.js runtime binary and execute it with any
+# parameters passed in.
+#
+# Set a variable named OSD_USE_NODE_JS_FILE_PATH to have it prefixed with OSD_HOME and executed
+# Example: SET OSD_USE_NODE_JS_FILE_PATH=\src\cli\dist
+#
+# NODE_OPTIONS is built using config/node.options and overridden by any previously set NODE_OPTIONS.
+# To pass in any specific defaults that can be overridden by both of them, use OSD_NODE_OPTS_PREFIX.
+
+SCRIPT="$0"
+
+UNAME=$(uname -s)
+if [ $UNAME = "FreeBSD" ]; then
+  OS="freebsd"
+elif [ $UNAME = "Darwin" ]; then
+  OS="darwin"
+else
+  OS="other"
+fi
+
+# SCRIPT may be an arbitrarily deep series of symlinks. Loop until we have the concrete path.
+while [ -h "$SCRIPT" ] ; do
+  ls=$(ls -ld "$SCRIPT")
```

Comment:
I would rather have another var for this, instead of same name in `ls`.

Reply:
That is ugly! i stole it from OS and didn't realize; will do.

Reply:
Done.